frontend/src/components/eventselect/reducer/AnalyticsEventSelect.tsx

```diff
@@ -68,7 +68,7 @@ const AnalyticsEventSelect: React.FC<EventsSelectProps> = (
     metadataUserAttributes,
     loading,
   } = props;
-
+  console.info('eventOptionList:', eventOptionList);
```

Comment:
Do we need it? or turn off info log by default?

Reply:
removed.